embedded/devices/src/rms.cpp

```diff
@@ -0,0 +1,292 @@
+#include "rms.h"
+
+#define TORQUE_SCALE_LWR(x) \
+  (((x)&0xFF) * 10) /* Converts Nm to the value the RMS wants */
+#define TORQUE_SCALE_UPR(x) (((x) >> 8) * 10)
+
+/* The following send functions are a series of cryptic steps
+ * that just magically make the RMS (Motor Controller) work.
+ * If you question them too hard you may burst into flames (along with the motor
+ * controller, so dont do it). The important part is that they
+ * are sent at the proper time and in the listed order. Timing information
+ * will be coming a little later */
+
+/* TODO figure out if heartbeat values may just be torque */
+
+RMS::RMS(CAN* c) : CANDevice(c) {}
+
+int RMS::begin() {
+  /*	initMotor();*/
+
+  // create CAN thread for this device
+  return 0;
+}
+
+int RMS::parser(uint32_t id, uint8_t* rmsData, uint32_t filter) {
+  if (filter != 0 && filter != id) {
+    return 1;
+  }
+  uint16_t val;
+  // uint16_t val2;
+  int16_t temp;
+  switch (id) {
+    case (0xa0):
```

Comment:
I'm not a huge fan of segments of abstract commented out code here. I understand the desire to keep it. I'm torn about this.

Could you at least add comments on what each ID carries in it's data bytes and we'll go from there as the season start?